assets/js/tooltips/hints-page.js

```diff
@@ -0,0 +1,128 @@
+// Dependencies
+// ------------
+
+import {extractTypeHint, extractModuleHint, extractFunctionHint} from './hints-extraction'
+import $ from 'jquery'
+import find from 'lodash.find'
+
+// Constants
+// ---------
+
+const contentInner = '.content-inner'
+const message = {hint: {}, ready: false, requestId: null}
+const typespecs = {
+  pathnameEnd: '/typespecs.html',
+  categories: [
+    { name: 'basicType', description: 'Basic type', hash: '#basic-types', detailsAvailable: false },
+    { name: 'literal', description: 'Literal', hash: '#literals', detailsAvailable: false },
```

Comment:
I think ExDoc doesn't link typespec literals, do you have any examples of this tooltip showing? 🙂

Reply:
> I think ExDoc doesn't link typespec literals, do you have any examples of this tooltip showing? 🙂

Ah, that makes sense!  I've not seen them anywhere, but added this category just in case, but I guess I can safely remove it. :)

Reply:
Given that the plan is to remove the typespec handling anyway, we don't have to worry about this particular bit either. :D